### PR TITLE
TINY-11115: Changed hex input field validation error text.

### PR DESCRIPTION
**Pull request by HAFRMO**
Related Ticket: TINY-11115

Description of Changes:
Wrong text, 

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ const english: Record<string, string> = {
   'colorcustom.rgb.hex.description': 'Hex color code',
   'colorcustom.rgb.range': 'Range 0 to 255',
   'colorcustom.rgb.invalid': 'Numbers only, 0 to 255',
+  'colorcustom.rgb.invalidHex': 'Hexadecimal only, 000000 to FFFFFF',
```
Comment (ztomaszyk) [inline modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts:25]: This should be also added to the `.pot` file

**Review by ztomaszyk — APPROVED** [review]

**Review by TheSpyder — APPROVED** [review]

File: modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts (hunk 1 above)
Comment (HAFRMO) [reply modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts:25]: That's not part of the dedicated translation issue?

Comment (tiny-ben-tran) [reply modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts:25]: Hmm.. Looks like [`Numbers only` is also missing in the translation](https://github.com/tinymce/tinymce-premium/blob/main/tools/language-packs/src/text/pot/core/tinymce_core.pot#L1407).

Comment (HAFRMO) [reply modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts:25]: Yes, they were never added to any translation thing, as that was a separate issue.

Comment (ztomaszyk) [reply modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts:25]: Hmm, ok. I've always added the new strings to `pot` files. With context to not have someone go through all of them again adding contexts. 
Do you have some additional ticket to add both errors to `pot` files?

**Pull request merged**